test/detect/webidl/default.txt

```diff
@@ -0,0 +1,24 @@
+/// Main namespace.
+namespace Namespace {
+    interface interface_identifier { /* interface_members... */ };
```

Comment:
What's the difference between a namespace and an interface? Both may include attributes and methods. Have they different bindings?